src/cmd/script.rs

```diff
@@ -45,109 +45,108 @@ pub fn new<'a, 'b>() -> App<'a, 'b> {
         )
         .arg(
-            Arg::with_name("vcom-arg")
+            Arg::new("vcom-arg")
                 .long("vcom-arg")
                 .help("Pass an argument to vcom calls (vsim/vhdlan/riviera only)")
                 .takes_value(true)
-                .multiple(true),
+                .multiple_values(true),
         )
         .arg(
-            Arg::with_name("vlog-arg")
+            Arg::new("vlog-arg")
                 .long("vlog-arg")
                 .help("Pass an argument to vlog calls (vsim/vlogan/riviera only)")
                 .takes_value(true)
-                .multiple(true),
+                .multiple_values(true),
         )
         .arg(
-            Arg::with_name("only-defines")
+            Arg::new("only-defines")
                 .long("only-defines")
                 .help("Only output commands to define macros (Vivado only)"),
         )
         .arg(
-            Arg::with_name("only-includes")
+            Arg::new("only-includes")
                 .long("only-includes")
                 .help("Only output commands to define include directories (Vivado only)"),
         )
         .arg(
-            Arg::with_name("only-sources")
+            Arg::new("only-sources")
                 .long("only-sources")
                 .help("Only output commands to define source files (Vivado only)"),
         )
         .arg(
-            Arg::with_name("no-simset")
+            Arg::new("no-simset")
                 .long("no-simset")
                 .help("Do not change `simset` fileset (Vivado only)"),
         )
         .arg(
-            Arg::with_name("vlogan-bin")
+            Arg::new("vlogan-bin")
                 .long("vlogan-bin")
                 .help("Specify a `vlogan` command")
                 .takes_value(true)
-                .multiple(false)
+                .multiple_values(false)
                 .default_value("vlogan")
                 .number_of_values(1),
         )
         .arg(
-            Arg::with_name("vhdlan-bin")
+            Arg::new("vhdlan-bin")
                 .long("vhdlan-bin")
                 .help("Specify a `vhdlan` command")
                 .takes_value(true)
-                .multiple(false)
+                .multiple_values(false)
                 .default_value("vhdlan")
                 .number_of_values(1),
         )
         .arg(
-            Arg::with_name("no-abort-on-error")
+            Arg::new("no-abort-on-error")
                 .long("no-abort-on-error")
                 .help("Do not abort analysis/compilation on first caught error (only for programs that support early aborting)")
         )
         .arg(
-            Arg::with_name("compilation_mode")
+            Arg::new("compilation_mode")
                 .long("compilation-mode")
                 .help("Choose compilation mode for Riviera-PRO option: separate/common (Riviera-PRO only)")
                 .takes_value(true)
-                .multiple(false)
```

Comment:
Did you remove this because it is the default?

Reply:
yes